pages/manage-contributions.js

```diff
@@ -167,13 +169,13 @@ class ManageContributionsPage extends React.Component {
                         {members.map(m => (
```

Comment:
:ice_cream:  Now that we're returning accounts, this should ideally be reflected on the name

```suggestion
                        {accounts.map(account => (
```